package.json

```diff
@@ -9,7 +9,7 @@
     "udp",
     "tcp"
   ],
-  "version": "0.4.5",
+  "version": "0.4.6",
```

Comment:
Please leave out the version bump, that'll be done as part of the release.

Reply:
cleanned.